rules_building_block/defense_evasion_unusual_process_extension.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2023/08/23"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/08/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies processes running with unusual extensions that are not typically valid for Windows executables.
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Unusual Process Extension"
+risk_score = 21
+rule_id = "1f460f12-a3cf-4105-9ebb-f788cc63f365"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Data Source: Elastic Defend", "Rule Type: BBR"]
+timestamp_override = "event.ingested"
+type = "eql"
+building_block_type = "default"
+
+query = '''
+process where host.os.type == "windows" and event.type == "start" and
+  process.executable : "?*" and not
+  process.name : ("*.exe", "*.com", "*.scr", "*.tmp", "*.dat") and
+  not process.executable : 
```

Comment:
Want to simplify this and avoid all exceptions and just look for MZ header and not standard extension? Sure, some will FP, but really its a BBR looking for _unusual_ ext. Could even just ignore all MS signed executables